sources/packages/api/src/utilities/datatable-config.ts

```diff
@@ -10,3 +10,12 @@ export enum FieldSortOrder {
  */
 export const DEFAULT_PAGE_LIMIT = 10;
 export const DEFAULT_PAGE_NUMBER = 0;
+
+// pagination option
+export class PaginationOptions {
+  searchName?: string;
```

Comment:
I believe that _searchName_ is too specific, can we rename it to _searchCriteria_?

Reply:
I believe this is the text coming from search input text. IMO.  searchText would be more self-descriptive.

Reply:
I believe that both are _self-descriptive_ 😉